core/templates/pages/contributor-dashboard-page/contributor-stats/contributor-stats.component.html

```diff
@@ -0,0 +1,601 @@
+<div class="stats-container">
+    <div>
```

Comment:
Each indentation level should be indented by only 2 spaces after the previous one.